pulp_smash/tests/pulp3/pulpcore/api_v3/test_crud_repos.py

```diff
@@ -43,8 +43,8 @@ def test_02_read_repo(self):
     def test_02_read_repos(self):
         """Search for the repository by its name.
 
-        Assert that just one search result is returned, and that that result
-        has a correct name.
+        Assert that just one search result is returned, and that result has a
```

Comment:
The point I was driving at in my previous comment is that this change shouldn't be made as part of this PR. This change is logically unrelated from the rest of this PR, and should be submitted as a separate PR.

Speaking of grammar, this change actually makes less grammatical sense. If you'd like to change this sentence to be less redundant, use a construct like "that the result has a correct name," not "that result has a correct name."